### DISCUSSION TITLE
Bluetooth gateway

### DISCUSSION THREAD

**Discussion by NuccioT** [indomus/forum#2478] [Hardware & Firmware]
Non mi è chiaro come funzioni un bluetooth gateway. In base alle informazioni acquisite dovrebbe essere un ponte fra un dispositivo bluetooth e la rete (e Home Assistant). Ma nel caso di un dispositivo con più sensori e possibilità di programmazione il tutto viene trasferito sull'hub?
Nel caso di specie vorrei utilizzare una valvola per irrigazione programmabile: da quello che ho visto su internet sono tutte bluetooth tranne una cinese (azzurrina flashabile tasmota che però richiede alimentazione via cavo).

Se io acquistassi un gateway bluetooth [come questo](https://amzn.to/3qcQS0T) e una valvola [come questa](https://amzn.to/426HHw7) riuscirei a comandare e programmare l'irrigazione via HA?
Grazie

**Comment by KC-inDomus** [comment]
Io bluetooth lo eviterei a prescindere. piuttosto zigbee, [con questa](https://indomus.it/componenti/woox-r7060-sgic-irrigatore-intelligente-zigbee/).

**Reply by nromagna** [reply]
Ciao @NuccioT, canalette permettendo, io ho messo dentro casa un Sonoff [come questo](https://www.amazon.it/Aggiornamento-Intelligente-Telecomando-Interruttore-Automazione/dp/B08C5DV434/ref=sr_1_6?keywords=sonoff+12+volt&qid=1685086144&sr=8-6) e sono arrivato a 2 normalissime elettrovalvole con 2 fili ciascuna...

**Reply by NuccioT** [reply]
Grazie ma purtroppo non è il mio caso. Devo collegarmi a un rubinetto che prende a stento la copertura wifi. Con un bridge cercavo di ridurre la distanza.

**Comment by NuccioT** [comment]
La soluzione indicata da KC è più percorribile se riesco a mettere un router zigbee esterno. Grazie

**Reply by 50494554524F** [reply]
parentesi: con homeassistant puoi usare schedine esp32 come ripetitori bluetooth
